# array of motor driver boards
MotorBoardInfo[] motorBoardInfo
